app/controllers/api/v1/api_token/api_token_api_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module ApiToken
+      class ApiTokenApiController < ApiController
```

Comment:
ApiController die include wel Knock::Authenticable, maar dat doet niks tenzij je zelf `authenticate_auth_user` (via isJwtAuthenticated) aanroept, toch? Of heeft dat nog andere rare side effects.

Reply:
nee volgens mij dat inderdaad.